index.js

```diff
@@ -67,7 +67,7 @@ bot.onText(/\/echo (.+)/, (msg, match) => {
 bot.onText(/\/heylisa/, msg => {
   //heylisa script
   var out =
-    "Hey, I am Lisa. I am neither related to Apple nor NASA. You can interact with me using the following commands:\n";
+    "Hey, I am odin. I am neither related to Apple nor NASA. You can interact with me using the following commands:\n";
```

Comment:
@waterval Please remove the `I am neither related to Apple nor NASA.` part too. Rest LGTM. Thanks.

Reply:
@sidntrivedi012 thank you for reviewing the correction. I have also removed the 'I am neither related to Apple nor NASA.' part as requested. Let me know if I can do anything else to help out

Reply:
@waterval LGTM. Thanks for contributing. 🙂 💯

Reply:
You're welcome. Thank you for merging the commit into master. Have a great day!